From 253e5bb70f1a13d2fc662f8df86e0aba52a82d11 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Mon, 7 Mar 2016 14:53:27 +0200
Subject: [PATCH 0606/1051] pd: Add emulation platform option

Used for A7K/A8K/A8K+ emulation plaform (Palladium)

Change-Id: Ia206e8f7f5c0562f8f6529126190dd52f4c5100a
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/Kconfig.platforms | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 21a715ad8222..3c2ff02d268a 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -35,6 +35,15 @@ config ARCH_BCM2835
 	  This enables support for the Broadcom BCM2837 SoC.
 	  This SoC is used in the Raspberry Pi 3 device.
 
+config ARCH_MVEBU_PD
+	bool "Marvell EBU SoC Emulation"
+	select ARCH_MVEBU
+	help
+	  This enables emulation support for Marvell EBU family, including:
+	   - Armada 3700 SoC Family
+	   - Armada 7K SoC Family
+	   - Armada 8K SoC Family
+
 config ARCH_BCM_IPROC
 	bool "Broadcom iProc SoC Family"
 	select COMMON_CLK_IPROC
-- 
2.17.1

